Client/Extensions/UIViewExtensions.swift

```diff
@@ -107,6 +107,28 @@ extension UIView {
         }
         return nil
     }
+
+    /// Shortcut to set the view's background colour to `.clear`, set the view's
+    /// `clipsToBounds` property set to true, and then add a blur effect on the view,
+    /// using the desired blur style.
+    ///
+    /// - Parameter style: The strength of the blur desired
+    func addBlurEffectWithClearBackgroundAndClipping(using style: UIBlurEffect.Style) {
+        self.clipsToBounds = true
+        self.backgroundColor = .clear
+        self.addBlurEffect(using: style)
+    }
+
+    /// Shortcut to set a blur effect on a view, given a specified style of blur desired.
+    ///
+    /// - Parameter style: The strength of the blur desired
+    func addBlurEffect(using style: UIBlurEffect.Style) {
+        let blurEffect = UIBlurEffect(style: style)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        blurEffectView.frame = self.bounds
```

Comment:
This will probably cause issues when bounds of the view are changing (rotation or split view on iPad). Could we use autolayout instead to prevent this?